src/client/src/orangehrmPerformancePlugin/components/ReviewSummary.vue

```diff
@@ -164,7 +164,9 @@ export default {
     const reviewDueDate = reviewDateFormat(props.dueDate);
 
     const employeeName = computed(() => {
-      return `${props.employee.firstName} ${props.employee.lastName} ${
+      return `${props.employee.firstName} ${
```

Comment:
can we use the `useEmployeeNameTranslate` here as well

Reply:
sure